### DISCUSSION TITLE
Thalamus performance issues

### DISCUSSION THREAD

**Discussion by Alpharius-30-milenium** [FakeFishGames/Barotrauma#13684] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

If wreck have 2+ thalamus brains you will get massive teleports if you playing multiplayer. It works even despite the fact that one of the brains seems to be dead anyway (but sometimes it's not dead, so, idk how it works)

### Reproduction steps

1. Create wreck with 2 or even more thalamus brains
2. Publish it/add to existing mod
3. Find that wreck
4. Suffer until you kill thalamus

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.1.18.1 (Treacherous Tides)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I'm afraid I could not reproduce this. Tested by adding 2 brains to the vanilla Kastrull, and I'm not seeing any unusual performance hit. It's worth noting though that normally you shouldn't add brains to the wrecks manually: one is spawned procedurally in a random hull (so you can't tell in advance where the brain is), and the extra ones will do nothing.

Would it be possible to share an example sub file? Could be that it has something to do with the placement of the brains.

**Comment by Alpharius-30-milenium** [comment]
Just to add context: I personally experienced this problem only with the biggest wrecks that consisted from 3 big ships but many people complaining that they was experiencing this problem with smaller wrecks. 
I will send all wrecks that currently represented in the mod in archive.
[Wreck.zip](https://github.com/Regalis11/Barotrauma/files/13179777/Wreck.zip)
